change/js/app.js

```diff
@@ -4,4 +4,11 @@ $(document).ready(function() {
 
   $("#time").text(Date.now());
 
+  //stuck :(
+
```

Comment:
Off to a good start.  You just need to figure out how to put your result on the page ;)